Translation of 'how-to-contribute' and 'css libraries...' (#3)
gatsbyjs/gatsby-es: docs/contributing/how-to-contribute.md

---
title: Cómo contribuir
---

> Queremos que contribuir a Gatsby sea divertido, agradable y educativo para cualquiera y para todos. Todas las contribuciones son bienvenidas, incluidas incidencias, nueva documentación y también actualizaciones, mejoras, artículos para el blog, workshops y más

Reviewer: ```suggestion
Queremos que contribuir a Gatsby sea divertido, agradable y educativo para cualquiera y para todos. Todas las contribuciones son bienvenidas, incluidas incidencias, nueva documentación y también actualizaciones, mejoras, artículos para el blog, talleres y más.
```

> Queremos que contribuir a Gatsby sea divertido, agradable y educativo para cualquiera y para todos. Todas las contribuciones son bienvenidas, incluidas incidencias, nueva documentación y también actualizaciones, mejoras, artículos para el blog, talleres y más.

Reviewer: ```suggestion
Queremos que contribuir a Gatsby sea divertido, agradable y educativo para cualquiera y para todos. Todas las contribuciones son bienvenidas, incluyendo issues, nueva documentación y también actualizaciones, mejoras, artículos para el blog, talleres y más.
```

@s-Hale Disculpa se me paso esta parte de los "issues" 😅 Por favor agrégala tambien 🙏

## ¿No estás seguro de cómo empezar a contribuir?

> Si está preocupado o no sabe por dónde empezar, verifica nuestra página [dónde participar](/contributing/where-to-participate/). Puedes ponerte en contacto con [Marcy Sutton (@marcysutton)](https://twitter.com/marcysutton) con preguntas o [@GatsbyJS](https://twitter.com/gatsbyjs) a Twitter, y cualquier del equipo de Gatsby en [Discord](https://gatsby.dev/discord). También puede [enviar un problema](/contributing/how-to-file-an-issue/) y un mantenedor puede darte orientación!

Reviewer: ```suggestion
Si estás preocupado o no sabes por dónde empezar, verifica nuestra página [dónde participar](/contributing/where-to-participate/). Puedes ponerte en contacto con [Marcy Sutton (@marcysutton)](https://twitter.com/marcysutton) con preguntas o [@GatsbyJS](https://twitter.com/gatsbyjs) en Twitter, y con cualquiera del equipo de Gatsby en [Discord](https://gatsby.dev/discord). También puedes [enviar un issue](/contributing/how-to-file-an-issue/) y un mantenedor puede darte orientación!
```

## Programación en pareja

> Gatsby.js ofrece sesiones gratis de programación para la comunidad, por si hay algo que te gustaría trabajar junto a un miembro del equipo. Ponte en contacto con nosotros si tienes una pregunta sobre cómo contribuir o alguna idea que quieras trabajar con alguno de nosotros!

Reviewer: ```suggestion
Gatsby.js ofrece sesiones gratis de [programación en parejas](/contributing/pair-programming/) para la comunidad, por si hay algo que te gustaría trabajar junto a un miembro del equipo. Ponte en contacto con nosotros si tienes una pregunta sobre cómo contribuir o alguna idea que quieras trabajar con alguno de nosotros!
```

---
title: Cómo contribuir
---

> Queremos que contribuir a Gatsby sea divertido, agradable y educativo para cualquiera y para todos. Todas las contribuciones son bienvenidas, incluidos temas, documentos nuevos así como las actualizaciones y los ajustes, entradas de blog, los talleres y más.

Reviewer: Pasando por acá, esta linea debería ser la 5, la línea número 4 debería ser una línea en blanco, por favor ponla nuevamente para mantener las mismas y evitar conflictos con git en el futuro 🚀

> 

Reviewer 2: Queremos que contribuir a Gatsby sea divertido, agradable y educativo para cualquiera y para todos. Todas las contribuciones son bienvenidas, incluidas incidencias, nueva documentación y también actualizaciones, mejoras, artículos para el blog, workshops y más.

Author: Gracias. Quiero aprender qué palabras tecnológicas se traducen al español y cuáles no.

## ¿No estás seguro de cómo empezar a contribuir?

Si está preocupado o no sabe por dónde empezar, verifica nuestra página [dónde participar](/contributing/where-to-participate/).

> Puedes contactar con preguntas a [Marcy Sutton (@marcysutton)](https://twitter.com/marcysutton) o [@GatsbyJS](https://twitter.com/gatsbyjs) a Twitter, y cualquier del equipo de Gatsby en [Discord](https://gatsby.dev/discord). También puede [enviar un problema](/contributing/how-to-file-an-issue/) y un mantenedor puede darte orientación!

Reviewer 2: Puedes ponerte en contacto con Marcy...

Author: y 'con preguntas' despues el enlace?

## Programación en pareja

> Gatsby.js ofrece [clases de programación en pareja](/contributing/pair-programming/) gratis a la comunidad,  si hay algo te gustaría trabajar juntos. Ponerse en contacto con nosotros si tienes una pregunta sobre cómo contribuir o una idea de algo para programación en pareja!

Reviewer 2: "Gatsby.js ofrece sesiones gratis de programación para la comunidad, por si hay algo que te gustaría trabajar junto a un miembro del equipo. Ponte en contacto con nosotros si tienes una pregunta sobre cómo contribuir o alguna idea que quieras trabajar con alguno de nosotros!"

que te parece así?

Author: Gracias! No estaba segura cómo de traducir 'pair programming'. Necesito practicar más la lectura y la escritura. Creo que todavía mi gramatica es muy inglés.

---
title: Cómo contribuir
---

Queremos que contribuir a Gatsby sea divertido, agradable y educativo para cualquiera y para todos. Todas las contribuciones son bienvenidas, incluidas incidencias, nueva documentación y también actualizaciones, mejoras, artículos para el blog, workshops y más

## ¿No estás seguro de cómo empezar a contribuir?

Si está preocupado o no sabe por dónde empezar, verifica nuestra página [dónde participar](/contributing/where-to-participate/).

> Puedes ponerte en contacto con [Marcy Sutton (@marcysutton)](https://twitter.com/marcysutton) con preguntas o [@GatsbyJS](https://twitter.com/gatsbyjs) a Twitter, y cualquier del equipo de Gatsby en [Discord](https://gatsby.dev/discord). También puede [enviar un problema](/contributing/how-to-file-an-issue/) y un mantenedor puede darte orientación!

Reviewer: Toda esta línea va continua con la línea 9, es decir, un solo parrafo.

> ## Programación en pareja

Reviewer: Esto debería ir en la línea 11.
